cli/packages/prisma-client-lib/src/utils/connectionNodeHasScalars.ts

```diff
@@ -0,0 +1,49 @@
+import {
+    GraphQLScalarType,
+    GraphQLEnumType
+} from 'graphql'
+
+import { getDeepType } from '.'
+
+function isScalar(field) {
+    const fieldType = getDeepType(field.type)
+
+    return (
+        fieldType instanceof GraphQLScalarType ||
+        fieldType instanceof GraphQLEnumType
+    )
+}
+
+export function connectionNodeHasScalars({ type }) {
+    const edgesField = Object.entries(type.getFields())
+        .filter(([, subField]: any) => {
+            return subField.name === 'edges'
+        })
```

Comment:
Please use `Object.values` here, then we can save the `.map` call https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values